share/wake/lib/system/job.wake

```diff
@@ -665,7 +665,7 @@ export topic runner: Runner
 publish runner =
     localRunner, defaultRunner, Nil
 
-def runJobImp label cmd env dir stdin res usage finputs foutputs vis pers run (LogLevel echo) (LogLevel stdout) (LogLevel stderr) isatty =
+export def runJobImp label cmd env dir stdin res usage finputs foutputs vis pers run (LogLevel echo) (LogLevel stdout) (LogLevel stderr) isatty =
```

Comment:
This is a bit more concerning to export. Is there a reason that you can't use `runJobWith localRunner` for execution needs? This exposes a bit more the internal functionality than I'm currently comfortable with.

Reply:
Yeah I can try to make it use `runJobWith`, though perhaps with the runner it chooses from the scoring

Reply:
I'm not sure I understand. I just meant I don't want to export runJobImp just yet (or if we export it we should think about it more). It seems to me that any external runner can be implemented in terms of `runJobWith localRunner`

Reply:
I think its easier for now if we export runJobImp, as it operates on the level of RunnerInput/Output rather than Plan (like runJobWith)

Reply:
Ok, that seems fine for now. I'll try to think of a better wrapper at some point.